src/screens/ProfileScreen/elements/BottomSheetImage.js

```diff
@@ -3,54 +3,56 @@ import {View, Text, StyleSheet, TouchableNativeFeedback, ActivityIndicator} from
 import {BottomSheet} from '../../../components/BottomSheet';
 import UserIcon from '../../../assets/icons/images/user.svg';
 import TrashIcon from '../../../assets/icons/images/trash.svg';
 import MediaIcon from '../../../assets/icons/images/media.svg';
 import CameraIcon from '../../../assets/icons/images/camera.svg';
+import {COLORS} from '../../../utils/theme';
 
 const BottomSheetImage = React.forwardRef((props, ref) => (
   <BottomSheet ref={ref} closeOnPressMask={true} height={300}>
     <View style={styles.containerBottomSheet}>
       <TouchableNativeFeedback onPress={() => props.onViewProfilePicture()}>
         <View style={styles.card}>
           <View style={styles.wrapCardImage}>
-            <UserIcon width={16.67} height={16.67} fill={colors.black} />
+            <UserIcon width={16.67} height={16.67} fill={COLORS.black} />
             <Text style={styles.textCard}>View profile picture</Text>
           </View>
         </View>
       </TouchableNativeFeedback>
       <TouchableNativeFeedback onPress={() => props.onOpenImageGalery()}>
         <View style={styles.card}>
           <View style={styles.wrapCardImage}>
-            <MediaIcon width={16.67} height={16.67} fill={colors.black} />
+            <MediaIcon width={16.67} height={16.67} fill={COLORS.black} />
             <Text style={styles.textCard}>Upload from library</Text>
           </View>
-          {props.isLoadingUpdateImageGallery ? (
-            <ActivityIndicator size="small" color="#0000ff" />
+          {props.isLoadingUpdateImageGalery ? (
+            <ActivityIndicator size="small" color={COLORS.darkBlue} />
           ) : null}
         </View>
       </TouchableNativeFeedback>
       <TouchableNativeFeedback onPress={() => props.onOpenCamera()}>
         <View style={styles.card}>
           <View style={styles.wrapCardImage}>
-            <CameraIcon width={16.67} height={16.67} fill={colors.black} />
+            <CameraIcon width={16.67} height={16.67} fill={COLORS.black} />
             <Text style={styles.textCard}>Take a photo</Text>
           </View>
           {props.isLoadingUpdateImageCamera ? (
-            <ActivityIndicator size="small" color="#0000ff" />
+            <ActivityIndicator size="small" color={COLORS.darkBlue} />
           ) : null}
         </View>
       </TouchableNativeFeedback>
 
       <TouchableNativeFeedback onPress={() => props.handleRemoveImageProfile()}>
         <View style={styles.card}>
           <View style={styles.wrapCardImage}>
-            <TrashIcon width={16.67} height={16.67} fill={colors.black} />
+            <TrashIcon width={16.67} height={16.67} fill={COLORS.black} />
             <Text style={styles.textCard}>Remove current picture</Text>
           </View>
-          {props.isLoadingRemoveImage ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+          {props.isLoadingRemoveImage ? (
+            <ActivityIndicator size="small" color={COLORS.darkBlue} />
+          ) : null}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the prop name `onOpenImageGalery` on line 24. It should be `onOpenImageGallery`. This could potentially cause issues if the parent component is passing the prop as `onOpenImageGallery`.
```diff
-        <TouchableNativeFeedback onPress={() => props.onOpenImageGalery()}>
+        <TouchableNativeFeedback onPress={() => props.onOpenImageGallery()}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->